src/Execute/ExecutionEnv.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from '../Backend/Command';
+
+interface ExecutionEnv {
```

Comment:
AFAIU, ExecutionEnv
- a target device: a local pc(simulator), a tv, ...
- This does not act(like install, run, uninstall, ...). This has info(Command) for the env. This is a kind of getter for Command of the specific env.
- This is a kind of data/info class for the specific env.
- This is a kind of CommandBuilder for the specific env.
- ExecutionEnv can